src/main/java/com/gregtechceu/gtceu/common/data/GTBlocks.java

```diff
@@ -565,70 +599,91 @@ private static BlockEntry<Block> createHermeticCasing(int tier) {
     }
 
     private static BlockEntry<Block> createSteamCasing(String name, String material) {
-        return REGISTRATE.block(name, Block::new)
+        var builder = REGISTRATE.block(name, Block::new)
                 .initialProperties(() -> Blocks.IRON_BLOCK)
                 .addLayer(() -> RenderType::cutoutMipped)
                 .blockstate(GTModels.createSteamCasingModel(name, material))
-                .tag(GTToolType.WRENCH.harvestTags.get(0), BlockTags.MINEABLE_WITH_PICKAXE)
-                .item(BlockItem::new)
+                .tag(GTToolType.WRENCH.harvestTags.get(0));
+        if (!ConfigHolder.INSTANCE.machines.requireGTToolsForBlocks) {
+            builder.tag(BlockTags.MINEABLE_WITH_PICKAXE);
+        }
+
+        return builder.item(BlockItem::new)
                 .build()
                 .register();
     }
 
     private static BlockEntry<CoilBlock> createCoilBlock(ICoilType coilType) {
-        BlockEntry<CoilBlock> coilBlock = REGISTRATE
+        var builder = REGISTRATE
                 .block("%s_coil_block".formatted(coilType.getName()), p -> new CoilBlock(p, coilType))
                 .initialProperties(() -> Blocks.IRON_BLOCK)
                 .properties(p -> p.isValidSpawn((state, level, pos, ent) -> false))
                 .addLayer(() -> RenderType::cutoutMipped)
                 .blockstate(GTModels.createCoilModel("%s_coil_block".formatted(coilType.getName()), coilType))
-                .tag(GTToolType.WRENCH.harvestTags.get(0), BlockTags.MINEABLE_WITH_PICKAXE)
-                .item(BlockItem::new)
+                .tag(GTToolType.WRENCH.harvestTags.get(0));
+        if (!ConfigHolder.INSTANCE.machines.requireGTToolsForBlocks) {
+            builder.tag(BlockTags.MINEABLE_WITH_PICKAXE);
+        }
```

Comment:
this won't work for removing the pickaxe tags as it's only applied when you run datagen.